### PR TITLE
merge 1.4

### PR DESCRIPTION
**Pull request by jmleroux**

### CODE REVIEW

File: CHANGELOG-1.3.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+# 1.3.26 (2015-09-16)
+
```
Comment (solivier) [inline CHANGELOG-1.3.md]: add at least a message

**Comment by skeleton** [comment]
GTM

File: src/Pim/Bundle/EnrichBundle/EventListener/Storage/ChannelLocaleSubscriber.php [hunk 2]
```diff
@@ -26,14 +28,22 @@ class ChannelLocaleSubscriber implements EventSubscriberInterface
...
+     * @param CompletenessGeneratorInterface $completeness
      */
-    public function __construct(LocaleRepositoryInterface $repository, BulkSaverInterface $saver)
-    {
-        $this->repository = $repository;
-        $this->saver      = $saver;
+    public function __construct(
+        LocaleRepositoryInterface $repository,
+        BulkSaverInterface $saver,
+        CompletenessGeneratorInterface $completeness = null
```
Comment (grena) [inline src/Pim/Bundle/EnrichBundle/EventListener/Storage/ChannelLocaleSubscriber.php]: Remove the default value on master, no?

File: src/Pim/Bundle/EnrichBundle/EventListener/Storage/ChannelLocaleSubscriber.php [hunk 3]
```diff
@@ -89,6 +99,10 @@ public function updateChannel(GenericEvent $event)
         foreach ($oldLocales as $locale) {
             $locale->removeChannel($channel);
             $updatedLocales[] = $locale;
+
+            if (null !== $this->completeness) {
```
Comment (grena) [inline src/Pim/Bundle/EnrichBundle/EventListener/Storage/ChannelLocaleSubscriber.php]: same

**Comment by ClementGautier** [comment]
GTM

**Comment by grena** [comment]
Comments to fix, then GTM

File: src/Pim/Bundle/EnrichBundle/EventListener/Storage/ChannelLocaleSubscriber.php (hunk 2 above)
Comment (ClementGautier) [inline src/Pim/Bundle/EnrichBundle/EventListener/Storage/ChannelLocaleSubscriber.php]: +1

File: src/Pim/Bundle/EnrichBundle/EventListener/Storage/ChannelLocaleSubscriber.php (hunk 3 above)
Comment (ClementGautier) [inline src/Pim/Bundle/EnrichBundle/EventListener/Storage/ChannelLocaleSubscriber.php]: +1

**Comment by grena** [comment]
(Oh, and there are some new commits on 1.4 to get back if possible. If not, it'll be for another PR :) )

**Comment by ClementGautier** [comment]
You have to remove the ChannelSaver as well and change the service definition to use the BaseSaver as the files are identical.

**Comment by ClementGautier** [comment]
GTM after the things on deprecated have been done

**Pull request merged**